tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -174,6 +174,21 @@ public function userLocksFileInProjectSpaceUsingWebDavAPI(string $user, string $
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Given user :user has locked file :file inside the space :space setting the following properties
+	 *
+	 * @param string $user
+	 * @param string $file
+	 * @param string $space
+	 * @param TableNode $properties table with no heading with | property | value |
+	 *
+	 * @return void
+	 */
+	public function userHasLockedFileInProjectSpaceUsingWebDavAPI(string $user, string $file, string $space, TableNode $properties): void {
+		$this->userLocksFileInProjectSpaceUsingWebDavAPI($user, $file, $space, $properties);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200, '');
```

Comment:
I would suggest this flow
```suggestion
		$response = $this->userLocksFileInProjectSpace($user, $file, $space, $properties);
		$this->featureContext->theHTTPStatusCodeShouldBe(200, '', $response);
```